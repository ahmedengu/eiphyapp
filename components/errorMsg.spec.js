import { mount } from 'enzyme';
import ErrorMsg from './errorMsg';

function setup(props = {}) {
  return mount(<ErrorMsg msg="this is an error" {...props} />);
}

describe('ErrorMsg Tests', () => {
  it('should work', () => {
    const actual = setup();
    expect(actual).toBe(actual);
  });

  it('should match snapshot', () => {
    const actual = setup();
    expect(actual.html()).toMatchSnapshot();
  });

  it('should return falsy with no message', () => {
    const actual = setup({ msg: '' });
    expect(actual.html()).toBeFalsy();
  });
});
