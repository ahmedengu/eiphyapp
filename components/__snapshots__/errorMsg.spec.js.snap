// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMsg Tests should match snapshot 1`] = `
<p>
  this is an error
</p>
`;
